app.ts

```diff
@@ -30,17 +30,23 @@ import url from 'url';
 
 import * as Sentry from '@sentry/node';
 import bodyParser from 'body-parser';
+// @ts-expect-warning
```

Comment:
what kind of error prompts this?

Reply:
`Could not find a declaration file for module 'compression'. '/home/ofeks/src/compiler-explorer/node_modules/compression/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/compression` if it exists or add a new declaration (.d.ts) file containing `declare module 'compression';`ts(7016)`

Reply:
does adding https://www.npmjs.com/package/@types/compression help? or same error?

Reply:
Another good catch! Thanks.  
I had trouble with adding @types/morgan (it doesn't play well with express) and dropped the additional types entirely, but other than morgan and systemd-socket (which has no @types) the rest work.  Fixed.